### ISSUE TITLE
Unifying teams

### ISSUE THREAD

**Issue by onmyway133** [hyperoslo/ios-foundation#69]
Currently we have many overlapping teams https://github.com/orgs/hyperoslo/teams. Should we unify them to avoid confusion ?

**Comment by zenangst** [comment]
Yeah I think this can be cleaned up. But this is a discussion that we should have on Slack rather on the team as it involves the entire tech team.
